src/bug-filing/azure-devops/azure-devops-bug-filing-service.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { isEmpty } from 'lodash';
+import * as React from 'react';
+
+import { EnvironmentInfo } from '../../common/environment-info-provider';
+import { NamedSFC } from '../../common/react/named-sfc';
+import { CreateIssueDetailsTextData } from '../../common/types/create-issue-details-text-data';
+import { BugServicePropertiesMap } from '../../common/types/store-data/user-configuration-store';
+import { BugFilingService } from '../types/bug-filing-service';
+import { SettingsFormProps } from '../types/settings-form-props';
+
+const AzureDevOpsBugFilingServiceKey = 'azureDevOps';
+
+export type AzureDevOpsBugFilingSettings = {
+    project: string;
+    issueDetailsLocation: string;
+};
+
+function buildStoreData(project: string, issueDetailsLocation: string): AzureDevOpsBugFilingSettings {
+    return {
+        project,
```

Comment:
looks like this is the project url... not just the project name (the placeholder shows: `https://dev.azure.com/org/project`)

Reply:
Will update.